code/modules/client/preference_setup/loadout/items/xeno/tajara.dm

```diff
@@ -42,6 +50,17 @@
 	boots["adhomian boots"] = /obj/item/clothing/shoes/workboots/tajara/adhomian_boots
 	gear_tweaks += new /datum/gear_tweak/path(boots)
 
+/datum/gear/shoes/tajara/recolourable_boots/New()
```

Comment:
move this to be beneath the main definition, you could move the one above as well

Reply:
This was added before I realized that auto-adapt sprites (the ones that taj uses) work on items that aren't in the xenowear tab; this code is functionless and I must've forgotten to remove it, I'll do that now.

I'm not sure what you mean by moving the list to the 'main definition' I could do that to the existing list if you need me to however I'm planning on removing my vestigial list.